tutorials/Application.EventEmitter.md

```diff
@@ -450,3 +455,69 @@ Generated when a child window starts loading.
     uuid: "454C7F31-A915-4EA2-83F2-CFA655453C52" // the UUID of the application
 }
 ```
+
+#### view-created
+Generated when a child BrowserView is created.
```

Comment:
We can drop "child Browserview" from this doc.

Reply:
done!